packages/hash/frontend/src/components/hooks/use-get-workspace-shortname-by-entity-uuid.ts

```diff
@@ -0,0 +1,75 @@
+import { extractBaseUri } from "@blockprotocol/type-system";
+import { systemUserShortname } from "@hashintel/hash-shared/environment";
+import { types } from "@hashintel/hash-shared/ontology-types";
+import { entityIdFromOwnedByIdAndEntityUuid } from "@hashintel/hash-subgraph";
+import { getRootsAsEntities } from "@hashintel/hash-subgraph/src/stdlib/element/entity";
+import { useEffect, useMemo, useState } from "react";
+import { useBlockProtocolGetEntity } from "./blockProtocolFunctions/knowledge/useBlockProtocolGetEntity";
+import { useGetWorkspaceByShortname } from "./use-get-workspace-by-shortname";
+
+export const useGetWorkspaceShortnameByEntityUuid = (params: {
```

Comment:
Same here re `get` in hook name.

Reply:
Addressed in https://github.com/hashintel/hash/pull/1650/commits/fa2ee8b97d56b079aca29075b602fd6243d6819b